regression/run.sh

```diff
@@ -257,14 +271,49 @@ elif [ "$1" = "issue_00502" ] ||\
 		expected_string="Executing FLATTEN pass (flatten design)."
 	elif [ "$1" = "issue_01223" ]; then
 		expected_string=" Executing CHECK pass (checking for obvious problems)."
-	elif [ "$1" = "issue_01231" ]; then
+	elif [ "$1" = "issue_01231" ] ||\
+		 [ "$1" = "issue_01364" ]; then
 		expected_string="Successfully finished Verilog frontend."
 	elif [ "$1" = "issue_01243" ]; then
 		expected_string="assign y = reg_assign;"
 	elif [ "$1" = "issue_01273" ]; then
 		expected_string="\$_MUX8_                         9"
 	elif [ "$1" = "issue_01329" ]; then
 		expected_string="\$mux                            1"
+	elif [ "$1" = "issue_01372" ]; then
+		expected_string="input \\[1:0\\] s"
+	elif [ "$1" = "issue_00329" ]; then
+		expected_string="wire \\[-1"
+	elif [ "$1" = "issue_00623" ]; then
+		expected_string="IOBUF _4_"
+	elif [ "$1" = "issue_00656" ]; then
+		expected_string="Successfully finished Verilog frontend."
+	elif [ "$1" = "issue_01014" ]; then
+		expected_string="Successfully finished Verilog frontend."
+	elif [ "$1" = "issue_01126" ]; then
+		expected_string="assign d = c\\[5:0\\]"
+	elif [ "$1" = "issue_01161" ]; then
+		expected_string="assign z0 = b"
```

Comment:
Can we move away from grepping for expected strings and start using `equiv_opt` and `select`?

Reply:
OK. Will update these tests in parts.

Reply:
Have updated some tests. But don't understand how to update tests where I need to check a particular connection as in issue_01161 or MSB for issue_00329. In the case with issue_01161 equiv_opt passes, but wires have wrong connections.